packages/ckeditor5-image/src/imageresize/resizeimagecommand.ts

```diff
@@ -77,4 +79,59 @@ export default class ResizeImageCommand extends Command {
 			} );
 		}
 	}
+
+	/**
+	 * Returns image width in specified units.
+	 *
+	 * 	* If image is not selected or command is disabled then `null` will be returned.
+	 * 	* If image is not fully loaded (and it is impossible to determine its natural size) then `null` will be returned.
+	 *  * If `targetUnit` percentage is passed then it will return width percentage of image related to its accessors.
+	 *
+	 * @param targetUnit Unit in which dimension will be returned.
+	 * @returns Parsed dimension with unit.
+	 */
+	public getSelectedImageWidthInUnits( targetUnit: string ): DimensionWithUnit | null {
```

Comment:
This method is stateless and should be generic in `utils/` as `getImageWidthInUnits()` (for any passed image, selected or not). It has nothing to do with the command.